code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -94,7 +94,23 @@
 				new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(src)
 				new /obj/item/weapon/storage/pill_bottle/hyperzine(src)
 				new /obj/item/weapon/card/id/syndicate(src)
-
+				
+			if("cowboy")
+				new /obj/item/clothing/accessory/storage/bandolier(src)
+					contents = list()
```

Comment:
Why is any of this indented, why are you resetting contents like this?

Are you trying to make this code be called on the new bandolier? Sadly that's not how DM works, what you should do is:

``` DM
var/obj/item/clothing/accessory/storage/bandolier/B = new /obj/item/clothing/accessory/storage/bandolier(src)
for (var/i = 1 to 6)
	new /obj/item/ammo_storage/speedloader/a357(B)

new /obj/item/clothing/suit/storage/det_suit(src)
...